node.js

```diff
@@ -663,160 +869,239 @@ async function stabilize() {
             // no need for handler
         }
     }
+
+    /* TBD 20191103 */
+    // update successor table - deviates from SIGCOMM
+    try {
+        await update_successor_table();
+    } catch (err) {
+        // probably no need for error handler
```

Comment:
Perhaps, but probably still best to conditionally log the error if we hit it. The implementation logic of update_successor_table could change in the future, in which case this error handling might matter